### PR TITLE
added github login to sandbox

### PR DESCRIPTION
**Pull request by shay23b**

### CODE REVIEW

File: src/const/apiEndpoints.js [hunk 1]
```diff
@@ -25,6 +25,7 @@ export const ApiEndpoints = {
     REMOVE_COMPANY_LOGO: '/usermgmt/removeCompanyLogo',
     EDIT_ANALYTICS: '/usermgmt/editAnalytics',
     GOOGLE_LOGIN: '/sandbox/login',
+    GITHUB_LOGIN: '/sandbox/login',
```
Comment (avrhamNeeman) [inline src/const/apiEndpoints.js:28]: Combine them into one prop: SANDBOX_LOGIN

File: src/domain/sandboxLogin/index.js [hunk 2]
```diff
@@ -41,15 +45,51 @@ const SandboxLogin = (props) => {
         username: '',
         password: ''
     });
-    const GOOGLE_CLIENT_ID = process.env.GOOGLE_CLIENT_ID;
+    // const GOOGLE_CLIENT_ID = process.env.GOOGLE_CLIENT_ID;
```
Comment (avrhamNeeman) [inline src/domain/sandboxLogin/index.js:48]: Remove comment

File: src/domain/sandboxLogin/index.js [hunk 3]
```diff
@@ -41,15 +45,51 @@ const SandboxLogin = (props) => {
...
+    const GOOGLE_CLIENT_ID = '916272522459-u0f4n2lh9llsielb3l5rob3dnt1fco76.apps.googleusercontent.com';
```
Comment (avrhamNeeman) [inline src/domain/sandboxLogin/index.js:49]: Move to const file

File: src/domain/sandboxLogin/index.js [hunk 4]
```diff
@@ -41,15 +45,51 @@ const SandboxLogin = (props) => {
...
+                false
+            );
+            AuthService.saveToLocalStorage(data);
+            history.push(referer);
+        } catch (ex) {
+            console.log(ex);
+        }
+    };
+
+    const handleGithubButtonClick = async () => {
```
Comment (avrhamNeeman) [inline src/domain/sandboxLogin/index.js:74]: async?

File: src/domain/sandboxLogin/index.js [hunk 5]
```diff
@@ -41,15 +45,51 @@ const SandboxLogin = (props) => {
...
         }
+        const url = window.location.href;
+        const shouldSigninWithGithub = url.includes('?code=');
+        const splittedUrl = url.split('?code=');
+        window.history.pushState({}, null, splittedUrl[0]);
+        if (shouldSigninWithGithub) {
+            if (splittedUrl.length > 1) {
+                signinWithGithub(`${splittedUrl[1]}`);
+            } else {
+                console.log('Authentication with GitHub failed');
```
Comment (avrhamNeeman) [inline src/domain/sandboxLogin/index.js:90]: Remove console.log

File: src/domain/sandboxLogin/index.js [hunk 6]
```diff
@@ -141,23 +182,29 @@ const SandboxLogin = (props) => {
...
-                            <div className="google-pad">
-                                <GoogleLogin
-                                    clientId={GOOGLE_CLIENT_ID}
-                                    className="google-login-button"
-                                    buttonText="Sign in with Google"
-                                    onSuccess={handleGoogleSignin}
-                                    onFailure={handleGoogleSignin}
-                                    cookiePolicy={'single_host_origin'}
-                                />
+                            <div>
```
Comment (avrhamNeeman) [inline src/domain/sandboxLogin/index.js:191]: Remove unnecessary tags

Comment (shay23b) [reply src/domain/sandboxLogin/index.js:191]: without this div the css goes crazy

**Review by idanasulinmemphis — APPROVED** [review]

**Pull request merged**